src/style.css

```diff
@@ -0,0 +1,31 @@
+body {
+  padding: 0;
```

Comment:
`body` doesn't have a default `padding`, so no need to write this.
also, you rewrite this rule on row №8